test/generators/generators.jl

```diff
@@ -1,12 +1,7 @@
-using CounterfactualExplanations
-using CounterfactualExplanations.Data
-using CounterfactualExplanations.Generators
-using Random
-
 @testset "Construction" begin
     @testset "Generic" begin
         generator = GenericGenerator()
```

Comment:
The generators could be called from Generators as well I believe? You've done this in a couple of places

Reply:
Right, will fix this